### DISCUSSION TITLE
Allow update pull requests from Github

### DISCUSSION THREAD

**Discussion by gotson** [xerial/sqlite-jdbc#748] [Ideas]
Hi @xerial, do you think you could enable this option in the repo ?

<img width="789" alt="Screenshot 2022-07-27 at 14 05 10" src="https://user-images.githubusercontent.com/2139133/181173244-b29d273e-372c-4def-be2a-0ebd1261f940.png">

That will enable rebasing pull requests by the maintainers from the PR screen, useful since we have a lot of old PRs at the moment.

**Comment by xerial** [comment]
Enabled
